app/router.js

```diff
@@ -74,6 +74,9 @@ Router.map(function() {
 
   if (!isEmpty(ENV.APP.ENABLE_PUBLICATIONS_TAB)) {
     this.route('publications', { path: '/publicaties', }, function() {
+      this.route('overview', { path: '/overzicht' }, function () {
+        this.route('all', { path: '/alle-dossiers' })
```

Comment:
Ik ben nog niet overtuigd van het gebruik van static routes. Ik denk dat dit vergetelheidjes in de hand gaat werken.